.pre-commit-config.yaml

```diff
@@ -27,6 +27,16 @@ repos:
           # Configure Black to support only syntax supported by the minimum supported Python version in setup.py.
           - --target-version=py37
         files: \.py$|\.pyi$
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
```

Comment:
Would it make sense to add an [isort config file](https://pycqa.github.io/isort/docs/configuration/config_files.html) that contains this configuration so that also IDEs can automatically pick this up?